CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ BUG FIXES:
 * Helm
   * Don't set TTL for server certificates when using Vault as the secrets backend. [[GH-1104](https://github.com/hashicorp/consul-k8s/pull/1104)]
   * Fix PodSecurityPolicies for clients/mesh gateways when hostNetwork is used. [[GH-1090](https://github.com/hashicorp/consul-k8s/pull/1090)]
+* CLI
+  * Fix `install` command for Windows. [[GH-1139](https://github.com/hashicorp/consul-k8s/pull/1139)]
```

Comment:
would this fix also work for other commands because the `chart.go` logic is common?

Reply:
Yes, it also fixes the `upgrade` command.

Reply:
updated!